notification.json

```diff
@@ -3,8 +3,8 @@
  "notifications": [
   {
    "id": 1,
-   "text": "Boostnote mobile app was released!",
-   "linkUrl": "https://github.com/boostio/boostnote-mobile"
+   "text": "Boostnote v0.8.15 is coming!",
```

Comment:
I prefer `Boostnote v0.8.15 was released` than that because `is coming` indicates it's not released.

Reply:
I don't understand the meaning of the sentence because the users who can see this message already have v0.8.15.

Reply:
Tell me about the person who know the all of updates at v0.8.15. We should the contents of update to users.

Reply:
how about `Update information` ?

Reply:
or `Update information v0.8.15` ?

Reply:
> Tell me about the person who know the all of updates at v0.8.15.

Don't say an impossible thing... It's impossible because we don't have ways to know all user's voice. However, there's an answer at least. It's us.

However, if `Update information v0.8.15`, your claim is correct.

Reply:
Before using this feature, we have to think any measurement means.

Reply:
In this case, the combination of https://medium.com/p/514692e76e1c/referrers and redirection(shorten url) is useful. If we use google (the url is https://goo.gl/7oTR2W), we can tell the access from Boostnote from other accesses.

Reply:
Great idea!👍 I will fix it like as you said.